libc/utils/gpu/loader/amdgpu/CMakeLists.txt

```diff
@@ -1,4 +1,11 @@
-add_executable(amdhsa-loader Loader.cpp)
+set(LLVM_LINK_COMPONENTS
+  BinaryFormat
+  Object
+  Option
+  Support
+  )
+
+add_llvm_executable(amdhsa-loader amdhsa-loader.cpp)
```

Comment:
will this be copied into `<build-dir>/bin`?

Reply:
Yes, it's also installed into `<install>/bin`.